artifacts/elsa-data-application-local-dev-test-docker-image/datasets.json5

```diff
@@ -0,0 +1,39 @@
+{
+  datasets: [
+    {
```

Comment:
Not sure if there is better data to use, or if the mocked aws data should be enabled as well?

Reply:
for dev (real aws) I think these are best for the moment..

but we can easily also put in the Smartie dataset and put it in dev bucket - as it is barely any data at all